test/files/stream.iced

```diff
@@ -0,0 +1,52 @@
+crypto = require('crypto')
+to_buf = require('../../src/stream-to-buffer.iced')
+stream = require('../../src/chunk-stream.iced')
+
+stream_random_data = (strm, len, cb) ->
+  written = 0
+  expected_results = []
+  while written < len
+    # generate random length
+    await crypto.randomBytes(1, defer(err, index))
+    if err then throw err
```

Comment:
Not a big deal since it's just test code, but it might make sense to use the same `cb(err, ret)` pattern that the functions you're calling are using. You can use the `esc` pattern to make this pretty convenient. The "error short-circuiter" type itself is a little trippy to follow, but examples of using it are all over our codebase and pretty easy to copy.